src/parser/parse_tree_cl.c

```diff
@@ -3103,6 +3103,27 @@ parser_append_node (PT_NODE * node, PT_NODE * list)
   return list;
 }
 
+/*
+ * parser_append_previous_node() -
+ *   return:
+ *   node(in/out):
+ *   list(in):
+ */
+PT_NODE *
+parser_append_previous_node (PT_NODE * node, PT_NODE * list)
+{
+  if (list)
+    {
+      node->next = list;
+      list = node;
+    }
+  else
+    {
+      list = node;
+    }
+  return list;
```

Comment:
It is shortened version.
```suggestion
  if (list)
    {
      node->next = list;
      node = list;
    }
  return node;
```

Reply:
I think it is not same.

Reply:
Assigning to the `list` variable changes the local variable, so there is no change outside this function.

Reply:
I wonder if you are intending to change the reference of the list variable passed when calling this function.

Reply:
`list` is return var. so I think it is okay.